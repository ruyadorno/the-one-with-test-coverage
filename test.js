const { log, notFound, readme } = require('./App')


describe('App', () => {
  it('should render readme', () => {
    expect(readme()).toMatchSnapshot()
  })
})
