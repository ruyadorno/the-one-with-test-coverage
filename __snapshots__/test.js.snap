// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render readme 1`] = `
"# the-one-with-test-coverage

Example repo that has a simplistic server implementation and multiple variations of tests around it.

The goal is to provide pratical examples of testing patterns that can help you improve the quality of teste coverage in your test suite.

"
`;
